src/connectors/copyparty.ts

```diff
@@ -0,0 +1,17 @@
+export {};
+
+Connector.playerSelector = '#np_inf';
+Connector.pauseButtonSelector = '#np_inf.playing';
+
+Connector.trackArtSelector = '#np_img';
+Connector.albumArtistSelector = '#np_circle';
+Connector.albumSelector = '#np_album';
+Connector.artistSelector = '#np_artist';
+Connector.trackSelector = '#np_title';
+
+Connector.currentTimeSelector = '#np_pos';
+Connector.durationSelector = '#np_dur';
+
+this.getUniqueID = () => {
```

Comment:
Sorry, missed this in my previous comment, but shouldn't this be `Connector.getUniqueID`?

Reply:
Hah, you're right! Guess that didn't do much huh...